src/renderer/components/ThemeSettings.vue

```diff
@@ -152,7 +158,13 @@ const baseThemeNames = computed(() => [
   t('Settings.Theme Settings.Base Theme.Gruvbox Dark'),
   t('Settings.Theme Settings.Base Theme.Gruvbox Light'),
   t('Settings.Theme Settings.Base Theme.Solarized Dark'),
-  t('Settings.Theme Settings.Base Theme.Solarized Light')
+  t('Settings.Theme Settings.Base Theme.Solarized Light'),
```

Comment:
Best to keep the theme groups sorted A-Z. Sorry normally i just mention to move it to a certain place without explaining it.

So sort all colors A-Z, Colors must be placed beneath Dracula Colors
Sort themes A-Z within the specified group, themes must be placed beneath Dracula theme. Keep the Hard-Medium-Low sorting within the theme